src/pvrclient-nextpvr.cpp

```diff
@@ -38,6 +39,8 @@
   #define MAXINT64 ULONG_MAX
 #endif 
 
+#include <algorithm>
```

Comment:
For what do you need this header?

Reply:
https://github.com/kodi-pvr/pvr.nextpvr/blob/0df87de8397c74f58e7371445677d088f67e2b55/src/pvrclient-nextpvr.cpp#L161

`algorithm` was included by `p8-platform/util/StdString.h`

Reply:
I hate people putting "using namespace std;" into sources. :-/